spec/nugets_pack_spec.rb

```diff
@@ -64,6 +64,13 @@ def self.create id, curr, gen_symbols = true
   end
 end
 
+describe Config, 'when setting #nuget_gem_exe' do
```

Comment:
You need to qualify: `Albacore::NugetsPack::Config` (if that's the location), otherwise, Ruby takes a closer `Config` class; that of rspec (I think it is).